opendatahub-io/odh-dashboard
# Add support to deployed server

## Description

Solves:
#648 
#759 
#659

## How Has This Been Tested?




## Testing instructions
1. Create a new dsg project
2. Create a new model server
3. Create a new deployed model
4. Check that both the model and the data connection are created
5. Go to Model Serving Global view
6. Deploy a new model
7. Click edit button
8. Edit the model

## Merge criteria:
